### PR TITLE
No track

### LINKED ISSUE #61: empty trackers component
image

### PR DESCRIPTION
**Pull request by Nourcova**
This page appear when there are no tracks

### CODE REVIEW

File: src/Pages/NoTrack/noTrack.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        <div className="flex flex-col items-center relative">
+          <img
+            src={Logo}
+            className="  bg-center bg-contain bg-white pb-20 px-16 z-0 mt-20 w-10/12 rounded-xl shadow-md"
+          />
+          <p className="text-xl z-10 absolute bottom-20 -m-4">
+            <b> No Trackers Yet!</b>
+          </p>
+          <div className="text-xs absolute -m-8  bottom-12">
+            <Button text="Add a new Tracker" />
```
Comment (JanaKhanji) [inline src/Pages/NoTrack/noTrack.js]: add onClick even if it only display an alert

File: src/Pages/NoTrack/noTrack.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Logo from '../../assets/images/NoTrack.png';
```
Comment (JanaKhanji) [inline src/Pages/NoTrack/noTrack.js]: call this "noTrack" instead of logo

**Review by JanaKhanji — CHANGES_REQUESTED** [review]

File: src/Pages/NoTrack/noTrack.js (hunk 1 above)
Comment (Nourcova) [reply src/Pages/NoTrack/noTrack.js]: OK

File: src/Pages/NoTrack/noTrack.js (hunk 2 above)
Comment (Nourcova) [reply src/Pages/NoTrack/noTrack.js]: Roger

**Review by osamaakb — CHANGES_REQUESTED** [review]
can you fix the extra files in this PR?

File: src/Pages/NoTrack/noTrack.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import notrack from '../../assets/images/NoTrack.png';
+import Button from '../../Components/button/button.js';
+function noTrack() {
```
Comment (JanaKhanji) [inline src/Pages/NoTrack/noTrack.js:4]: Add a prop onClick to be called when the btn is clicked

**Review by JanaKhanji — CHANGES_REQUESTED** [review]

**Pull request merged**